src/pages/EnablePayments/IdologyQuestions.js

```diff
@@ -109,7 +109,6 @@ class IdologyQuestions extends React.Component {
                         idNumber: this.props.idNumber,
                     },
                 });
-                return {answers, isLoading: true};
```

Comment:
Do we still need to return `answers` here?

Reply:
Probably. This change should not be done in the PR anyways, so removing it. Thanks!